test/known_issues/test-vm-getters.js

```diff
@@ -16,4 +16,9 @@ const context = vm.createContext(sandbox);
 const code = 'Object.getOwnPropertyDescriptor(this, "prop");';
 const result = vm.runInContext(code, context);
 
-assert.strictEqual(result, descriptor);
+// Ref: https://github.com/nodejs/node/issues/11803
+
+assert.deepStrictEqual(Object.keys(result), Object.keys(descriptor));
+for (const prop of Object.keys(result)) {
+  assert.strictEqual(result[prop], descriptor[prop]);
+}
```

Comment:
Have you checked whether using `assert.deepStrictEqual(Object.assign({}, result), descriptor)` instead works too? I am not a 100 % sure it’s more readable so feel free to ignore me. :smile: